test/fork.js

```diff
@@ -41,12 +41,17 @@ test('rejects on error and streams output', function (t) {
 });
 
 test('exit after tests are finished', function (t) {
-	t.plan(1);
+	t.plan(2);
 
 	var start = Date.now();
+	var cleanupCompleted = false;
 
 	fork(fixture('long-running.js'))
 		.on('exit', function () {
-			t.ok(Date.now() - start < 10000);
+			t.ok(Date.now() - start < 10000, 'did NOT wait for setTimeout(fn, 15000');
+			t.ok(cleanupCompleted, 'did wait for onExit(fn) to complete');
```

Comment:
@jamestalmage I've added Windows testing with AppVeyor and this assertion fails: https://ci.appveyor.com/project/sindresorhus/ava/build/2/job/w7efao2if7n2g2q9 Could you take a look?